### ISSUE TITLE
Will marktext support css?

### ISSUE THREAD

**Issue by JamesWZM** [marktext/marktext#3379]
It seems that MarkText only support HTML without css

![image](https://user-images.githubusercontent.com/48663928/182331678-1bcd9b64-ef73-4a87-a3e4-d138be5b0f03.png)

![image](https://user-images.githubusercontent.com/48663928/182331620-6cd29211-af0b-482d-865e-5afe5a781b20.png)

**Comment by Jocs** [comment]
Yes, MarkText supports custom CSS via the **`customCss`** field in **Preferences > Theme**. You can inject any CSS rules to style the editor and exported content. Closing as answered.
